app_legacy/Helpers/render/layout.php

```diff
@@ -38,73 +38,92 @@ function RenderToolbar(): void
-    echo "<li><a href='/gameList.php?c=13'>Atari Lynx</a></li>";
-    // NEC
-    echo "<li class='dropdown-header'>NEC</li>";
-    echo "<li><a href='/gameList.php?c=8'>PC Engine/TurboGrafx-16</a></li>";
-    echo "<li><a href='/gameList.php?c=76'>PC Engine CD/TurboGrafx-CD</a></li>";
-    echo "<li><a href='/gameList.php?c=47'>PC-8000/8800</a></li>";
-    echo "<li><a href='/gameList.php?c=49'>PC-FX</a></li>";
-    echo "</ul>";
+    foreach ($menuSystemsList as $column){
+        echo "<ul>";
+        foreach ($column as $brand => $systems) {
+            echo "<li class='dropdown-header'>$brand</li>";
+            foreach ($systems as $system){
 
-    echo "<ul>";
-    // Sega
-    echo "<li class='dropdown-header'>Sega</li>";
-    echo "<li><a href='/gameList.php?c=33'>SG-1000</a></li>";
-    echo "<li><a href='/gameList.php?c=11'>Master System</a></li>";
-    echo "<li><a href='/gameList.php?c=15'>Game Gear</a></li>";
-    echo "<li><a href='/gameList.php?c=1'>Genesis/Mega Drive</a></li>";
-    echo "<li><a href='/gameList.php?c=9'>Sega CD</a></li>";
-    echo "<li><a href='/gameList.php?c=10'>Sega 32X</a></li>";
-    echo "<li><a href='/gameList.php?c=39'>Sega Saturn</a></li>";
-    echo "<li><a href='/gameList.php?c=40'>Sega Dreamcast</a></li>";
-    // Other
-    echo "<li class='dropdown-header'>Other</li>";
-    echo "<li><a href='/gameList.php?c=43'>3DO Interactive Multiplayer</a></li>";
-    echo "<li><a href='/gameList.php?c=37'>Amstrad CPC</a></li>";
-    echo "<li><a href='/gameList.php?c=38'>Apple II</a></li>";
-    echo "<li><a href='/gameList.php?c=27'>Arcade</a></li>";
-    echo "<li><a href='/gameList.php?c=71'>Arduboy</a></li>";
-    echo "<li><a href='/gameList.php?c=44'>ColecoVision</a></li>";
-    echo "<li><a href='/gameList.php?c=57'>Fairchild Channel F</a><li>";
-    echo "<li><a href='/gameList.php?c=45'>Intellivision</a></li>";
-    echo "<li><a href='/gameList.php?c=23'>Magnavox Odyssey 2</a></li>";
-    echo "<li><a href='/gameList.php?c=69'>Mega Duck</a></li>";
-    echo "<li><a href='/gameList.php?c=29'>MSX</a></li>";
-    echo "<li><a href='/gameList.php?c=14'>Neo Geo Pocket</a></li>";
-    echo "<li><a href='/gameList.php?c=46'>Vectrex</a></li>";
-    echo "<li><a href='/gameList.php?c=72'>WASM-4</a></li>";
-    echo "<li><a href='/gameList.php?c=63'>Watara Supervision</a></li>";
-    echo "<li><a href='/gameList.php?c=53'>WonderSwan</a></li>";
-    echo "</ul>";
+                $systemName = $system['systemName'];
+                $listId = $system['listID'];
+                $cleanSystemShortName = Str::lower(str_replace("/", "", config("systems.$listId.name_short")));
+                $iconName = Str::kebab($cleanSystemShortName);
+                echo "<li><a href='/gameList.php?c=$listId' class='!flex items-center gap-x-2' >"; // the flex class needs to be forced here
+                echo " <img src='assets/images/system/$iconName.png' width='16' height='16' alt=''>";
```

Comment:
This should be `asset("assets/images/system/$iconName.png")` to make sure image URLs are prefixed with the configured static host.